templates/senator/_layout.html.twig

```diff
@@ -73,6 +73,12 @@
                             </li>
                             {% endif %}
 
+                            {% if is_granted('ROLE_SENATOR') %}
```

Comment:
I see we check `is_granted('ROLE_SENATOR')` many times in this template. 
Could you refactor this for checking once this role, please?

Reply:
fixed :)

Reply:
the best will be: 
```twig
{% if is_senator %}
  <li>ITEM 1</li>
{% endif %}

{% if is_senator %}
  <li>ITEM 2</li>
{% endif %}
```
=> 
```twig
{% if is_senator %}
  <li>ITEM 1</li>
  <li>ITEM 2</li>
{% endif %}
```

Reply:
indeed, re fixed